images/scripts/rhel.setup

```diff
@@ -156,6 +156,7 @@ ltrace \
 nmap-ncat \
 python3-tracer \
 qemu-kvm \
+redis \
```

Comment:
No need to do the same?

https://github.com/cockpit-project/bots/commit/16ec1980bdc4b333ec2de793a85eedd1dd7511e3#diff-b0928b5227c29cd0b5ef960d71d5bb357c1f1831de3a3a5148b74910386ad86dR191

Reply:
No, services don't auto-start by default in Fedora/RHEL.

Reply:
... in fact, that cost me half an hour of my life as when it's not running, the pmproxy API requests hang forever :grin:

Reply:
interesting - I had no idea